ui/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
-    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
+    "@kubernetes/client-node": "github:scality/kubernetes-client-javascript.git#browser-dist",
```

Comment:
Don't really like this TBH.

Reply:
Me neither. https://github.com/scality/metalk8s/pull/782#discussion_r267705252
=> debt ticket